merge_changelogs.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
```

Comment:
Ensure to delete this temporary script before merging to main.

Reply:
Addressed by [922be49](https://github.com/omnivector-solutions/jobbergate/pull/690/commits/922be4979eede892caddd8b7ad19b9065d468351).